### DISCUSSION TITLE
Can we create multi platform plugin with Kotlin?

### DISCUSSION THREAD

**Discussion by MustafaEminn** [ionic-team/capacitor#6049] [Q&A]
Hello, Kotlin has multi platform feature. If I develop some Capacitor plugin in Kotlin language, can I also create iOS plugin from kotlin plugin?

**Comment by jcesarmobile** [comment]
It's most likely not possible since Capacitor plugins rely on .m files for declaring the plugin methods. But have never looked at Kotlin multiplatform, so I could be wrong.
